apps/docs/content/components/CTABanner.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: CTABanner
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=539%3A14285&t=RJEWZhVUtrRQ0pKh-0'
+source: https://github.com/primer/brand/tree/main/src/CTABanner/CTABanner.tsx
+storybook: '/brand/storybook/?path=/docs/components-ctabanner--playground'
+description: Use CTABanner to create call to action content in a separated section.
+---
+
+```js
+import {CTABanner, ButtonGroup, Button} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<CTABanner>
+  <CTABanner.Heading>
+    Where the most ambitious teams build great things
+  </CTABanner.Heading>
+  <CTABanner.Description>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
+    ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus
+    elementum.
+  </CTABanner.Description>
+  <ButtonGroup buttonSize="large">
```

Comment:
should `buttonSize` be configurable @danielguillan? If not, i'd recommend we set this by default inside CTABanner. Buttons feel a bit large to me, so wondering if it should be `medium` too.

Reply:
When I was working with Ash he said to use the large buttons medium is set by default. This is why it was created this way.

Reply:
> When I was working with Ash he said to use the large buttons medium is set by default. This is why it was created this way.

That's fine, if large is default, can you set it _inside_ the component and remove from the docs examples? This is part of design logic, so you'd also probably want a test around that. I'd expect the only time you'd need to manually the buttonSize prop, is to change it to 'medium'.